### DISCUSSION TITLE
Should BBL Cover blinds open completely?

### DISCUSSION THREAD

**Discussion by danst0** [SukramJ/aiohomematic#1570] [General]
As demanded in issues #1552 and #1560 by @SukramJ  I am opening the discussion if BBL blinds should open completely. 
Tldr; The answer is of course, yes.

I am not able to fully open my raffstore blinds with Homematic bbl switches from home assistant anymore.

behavior in CCU: press the open button --> blinds open completely
behavior in home assistant: press the open button --> blinds open completely, then go down a little. It does not help to first call the open tilt command or to manually set the tilt to 100% either before or after moving up.
In the CCU, the tilt is shown as 50%, wtf?

I am not too much into programming so it would at least be great to have a workaround, that I could implement.

**Comment by SukramJ** [comment]
Danke das du dich der Sache angenommen und eine Diskussion gestartet hast.
Das Beitrag der Meisten ist leider So, das bei einer Aufforderung mal selbst aktiv zu werden und einen Beitrag zu leisten, leider nichts mehr kommt. 
Das ursprüngliche Verhalten der Lammellen wird mit der nächsten Version wieder hergestellt.

**Reply by danst0** [reply]
Cool, danke!
